crate/cli/src/actions/shared/delete_attributes.rs

```diff
@@ -0,0 +1,89 @@
+use clap::Parser;
+use cosmian_kms_client::{
+    cosmian_kmip::kmip::kmip_types::UniqueIdentifier,
+    kmip::{
+        kmip_operations::{DeleteAttribute, DeleteAttributeResponse},
+        kmip_types::{Attribute, AttributeReference, Tag},
+    },
+    KmsClient,
+};
+use tracing::trace;
+
+use super::set_attributes::SetOrDeleteAttributes;
+use crate::{actions::console, cli_bail, error::result::CliResult};
+
+/// Delete the KMIP object attributes (one or multiple attributes).
+#[derive(Parser, Debug)]
+#[clap(verbatim_doc_comment)]
+pub struct DeleteAttributesAction {
+    #[clap(flatten)]
+    requested_attributes: SetOrDeleteAttributes,
+
+    /// The attributes or tags to retrieve.
+    /// To specify multiple attributes, use the option multiple times.
+    #[clap(long = "attribute", value_name = "ATTRIBUTE", verbatim_doc_comment)]
+    attribute_tags: Option<Vec<Tag>>,
+}
+
+impl DeleteAttributesAction {
+    async fn delete_attribute(
```

Comment:
struct is plural - functions are not for `attributes`
Is this OK?

Reply:
This time, it is singular (since ckms sends as many POST requests as there are attributes to be deleted)